.travis.yml

```diff
@@ -4,12 +4,15 @@ sudo: required
 language: bash
 services:
   - docker
+git:
+  depth: false
 
 before_install:
   - curl -Ls -o docker-build https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/docker-build
   - chmod +x docker-build
 
 install:
+  - git fetch --tags
```

Comment:
To my mind this is more of a `before_install` command.

Reply:
Good call, I was referencing a project that didn't have a `before_install` section at all, but that does make more sense in this case.